integration-tests/integration/groups.spec.js

```diff
@@ -152,12 +158,33 @@ context('Groups', () => {
       const groupDetails = GroupDetailsPage.verifyOnPage('Site 1 - Group 2')
       groupDetails.deleteGroupNoChild().should('not.exist')
 
-      cy.visit('/manage-groups/SITE_1_GROUP_2/delete-group', { failOnStatusCode: false })
+      cy.visit('/manage-groups/SITE_1_GROUP_2/delete/children/none', { failOnStatusCode: false })
       AuthErrorPage.verifyOnPage()
     })
 
+    it('Can access delete group directly if authorised', () => {
```

Comment:
I have added this test to identify that the Group Delete Page is accessed via '/manage-groups/<group-name>/delete/children/none'